tests/xilinx_test.sh

```diff
@@ -57,6 +57,7 @@ run_all() {
     # Args:
     #  0: Boolean flag that runs all (1) or a reduced set (0) of samples
     run_sample remove_degenerate_loop remove_degenerate_loop_test
+    run_sample pipeline_test pipeline_test 
```

Comment:
no shift register test for Xilinx

Reply:
This is intentional, we are not implementing it for Xilinx now